pkg/gds/certman/certs_test.go

```diff
@@ -312,6 +313,36 @@ func (s *certTestSuite) TestCertManagerSevenDayReissuanceReminder() {
 	emails.CheckEmails(s.T(), messages)
 }
 
+func (s *certTestSuite) TestCertManagerExpiration() {
+	s.setupCertManager(sectigo.ProfileCipherTraceEE, fixtures.Small)
+	defer s.teardownCertManager()
+	defer s.fixtures.LoadReferenceFixtures()
+	require := s.Require()
+
+	// setup the datastore to contain the modified charlieVASP
+	hotelVASP, err := s.fixtures.GetVASP("hotel")
+	require.NoError(err, "could not get hotel VASP")
+	hotelVASP = s.setupVASP(hotelVASP)
+
+	certID := certman.RetrieveCertID(hotelVASP.IdentityCertificate)
+	cert := &models.Certificate{
+		Id:     certID,
+		Status: models.CertificateState_ISSUED,
+	}
+	err = s.db.UpdateCert(cert)
+	require.NoError(err)
+
+	// Run the loop again to ensure that emails are not resent to contacts
+	s.certman.HandleCertificateReissuance()
+
+	cert, err = s.db.RetrieveCert(certID)
+	require.NoError(err)
+	require.Equal(cert.Status, models.CertificateState_EXPIRED)
```

Comment:
So `hotelVASP` has an expired certificate by default?

Reply:
yep, see the above comment.